WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java

```diff
@@ -333,17 +335,28 @@ private void refreshBlogDetails(@Nullable final Blog blog) {
         }
 
         if (blog == null) {
-            mScrollView.setVisibility(View.GONE);
-            mFabView.setVisibility(View.GONE);
-            mNoSiteView.setVisibility(View.VISIBLE);
-
-            // if the screen height is too short, we can just hide the drake illustration
-            Activity activity = getActivity();
-            boolean drakeVisibility = DisplayUtils.getDisplayPixelHeight(activity) >= 500;
-            if (drakeVisibility) {
-                mNoSiteDrakeImageView.setVisibility(View.VISIBLE);
+            showNoSitesView();
+
+            return;
+        } else if (blog.getAutomatedTransfer()) {
+            showNoSitesView();
+
+            int numBlogs = WordPress.wpDB.getNumBlogs();
+            WPTextView title = (WPTextView) getView().findViewById(R.id.my_site_no_site_title);
+            WPTextView prompt = (WPTextView) getView().findViewById(R.id.my_site_no_site_prompt);
+            title.setText("Your site is not supported in this version of the app. Please update your app in the Play Store");
```

Comment:
I know it's a temporary hack, but i18n matters (for this string and strings below).